image/mediatypes.go

```diff
@@ -0,0 +1,13 @@
+package image
+
+// mediatype definitions for image components handled in containerd.
+//
+// oci components are generally referenced directly, although we may centralize
+// here for clarity.
+const (
+	MediaTypeDockerSchema2Layer        = "application/vnd.docker.image.rootfs.diff.tar"
```

Comment:
Do we expect the media types to be static and common for all `Resolver`s/`Fetcher`s or is this just a temporary set of constants?